tests/phpunit/tests/test-wpml-elementor-translate-ids.php

```diff
@@ -48,14 +45,21 @@ public function it_should_translate_template_id() {
 
 		\WP_Mock::onFilter( 'wpml_object_id' )
 		        ->with( $template_id, $post_type, true )
-		        ->reply( $translated_id );
+		        ->reply( (int) $translated_id );
 
 		$subject = new WPML_Elementor_Translate_IDs( \Mockery::mock( '\WPML\Utils\DebugBackTrace' ) );
 
 		$this->assertEquals( $translated_id, $subject->translate_theme_location_template_id( $template_id ) );
 
 	}
 
+	public function dp_translate_template_id() {
+		return [
+			[ 123, 456, 'anything' ],
+			[ '123', '456', 'anything' ],
+		];
```

Comment:
@dgwatkins I would add keys so it's clear why these two a different.
```php
return [
	'preserve integer type' => [ 123, 456, 'anything' ],
	'preserve string type'   => [ '123', '456', 'anything' ],
];
```

Reply:
I handled in https://github.com/OnTheGoSystems/wpml-page-builders-elementor/pull/141 because this was already merged when I got here.